shell/platform/windows/flutter_windows_view.cc

```diff
@@ -8,6 +8,19 @@
 
 namespace flutter {
 
+/// Returns true if the surface will be updated as part of the resize process.
```

Comment:
Can you flesh this out to specifically say what other code this is required to stay in sync with?